test/plugin/negativeYear.test.js

```diff
@@ -0,0 +1,54 @@
+import moment from 'moment'
+import MockDate from 'mockdate'
+import dayjs from 'dayjs'
+import negativeYear from '../../src/plugin/negativeYear'
+import utc from '../../src/plugin/utc'
+import { REGEX_PARSE } from '../../src/constant'
+
+
+dayjs.extend(negativeYear)
+dayjs.extend(utc)
+
+beforeEach(() => {
+  MockDate.set(new Date())
+})
+
+afterEach(() => {
+  MockDate.reset()
+})
+
+describe('negativeYear', () => {
+  it('parses negative years', () => {
+    expect(dayjs('-2020-01-01').year()).toBe(-2020)
+    const date = '-2021/01/03'
+    const date2 = '01/03/-2021'
+    const date3 = '01-03--2021'
+    const d = date.match(REGEX_PARSE)
+    expect(dayjs(date).format('YYYY-MM-DD')).toBe(`-${moment(date).format('YYYY-MM-DD')}`)
+    expect(dayjs(date).format()).toBe(`-${moment(date).format()}`)
```

Comment:
we could remove moment in this test case, as far as i know, moment does not support nagative year

Reply:
@iamkun Sure